### ISSUE TITLE
sinh with core.matrix

### ISSUE THREAD

**Issue by akhudek** [mikera/vectorz-clj#11]
``` clojure

  (set-current-implementation :double-array)
  (pm (sinh (new-vector 10)))
  [0.000 0.000 0.000 0.000 0.000 0.000 0.000 0.000 0.000 0.000]

  (set-current-implementation :vectorz)
  ;; pm doesn't work with this
  (sinh (new-vector 10))
  #<Vector [1.0,1.0,1.0,1.0,1.0,1.0,1.0,1.0,1.0,1.0]>
```

**Comment by mikera** [comment]
Thanks for the report!

Yes, confirmed this is a bug in Vectorz (it's actually computing cosh by mistake.......). Clearly nobody has been testing the hyperbolic functions much :-)

I'll push a new Vectorz version that fixes this in the near future.

**Comment by mikera** [comment]
This is fixed in the vectorz-clj 0.18.0 release

**Comment by akhudek** [comment]
Great, thanks!

On December 22, 2013 at 5:32:26 PM, Mike Anderson (notifications@github.com) wrote:

This is fixed in the vectorz-clj 0.18.0 release
